R/scalevariableset.R

```diff
@@ -103,6 +106,8 @@ replaceFactorsWithValues <- function(x)
 #' @noRd
 removeSUMColumns <- function(df)
 {
+    if (NCOL(df) == 1) # No SUM column if 
+        return(df)
```

Comment:
Seems reasonable that a single column data frame won't have a data reduction column but the comment here is incomplete?